django/core/management/templates.py

```diff
@@ -76,12 +75,8 @@ def handle(self, app_or_project, name, target=None, **options):
             top_dir = path.join(os.getcwd(), name)
             try:
                 os.makedirs(top_dir)
-            except OSError as e:
-                if e.errno == errno.EEXIST:
-                    message = "'%s' already exists" % top_dir
-                else:
-                    message = e
-                raise CommandError(message)
+            except FileExistsError:
+                raise CommandError("'%s' already exists" % top_dir)
```

Comment:
Correct, I'm not sure if the behavior change matters or what other `OSError` might happen. If you think I should spend time trying to improve test coverage here, I'll do that.